client/src/constants/Main/lotteryEventData.ts

```diff
@@ -0,0 +1,34 @@
+import { LotteryEventProps } from "@/components/LotteryEvent";
+
+export const lotteryEventData: LotteryEventProps[] = [
```

Comment:
```suggestion
export const LOTTERY_EVENT_DATA: LotteryEventProps[] = [
```

상수 네이밍은 대문자 snake case가 더 상수느낌나고 좋지 않을까욥?

Reply:
반영했습니다 온니😚